internal/impl/io/input_websocket.go

```diff
@@ -176,9 +178,27 @@ func (w *websocketReader) Connect(ctx context.Context) error {
 	}
 
 	if len(w.openMsg) > 0 {
-		if err := client.WriteMessage(openMsgType, w.openMsg); err != nil {
-			return err
+		if openMsgType == websocket.BinaryMessage {
+			if err := client.WriteMessage(openMsgType, w.openMsg); err != nil {
+				return err
+			}
+		} else if openMsgType == websocket.TextMessage {
+			openMessag := string(w.openMsg)
+			messages := strings.Split(openMessag, "\n")
```

Comment:
What if I want to send a single multiple text? e.g. I'd like to send a single message to server with payload
```
first_line
second_line
```

Instead of two messages
MSG1:
```
first_line
```
MSG2:
```
second_line
```

Reply:
we cannot do that, only single line supported now